src/Silex/WebTestCase.php

```diff
@@ -42,14 +41,14 @@ public function setUp()
     /**
      * Creates the application.
      *
-     * @return HttpKernel
+     * @return Application
```

Comment:
You can return any `HttpKernel` (like the `HttpCache` or any StackPHP middleware).